src/status_im/ui/screens/profile/user/styles.cljs

```diff
@@ -1,6 +1,80 @@
-(ns status-im.ui.screens.profile.user.styles)
+(ns status-im.ui.screens.profile.user.styles
+  (:require [quo2.foundations.colors :as colors]
+            [react-native.safe-area :as safe-area]))
 
 (def share-link-button
   {:margin-top        12
    :margin-horizontal 16
    :margin-bottom     16})
+
+(def radius 16)
+
+(def top-background-view
+  {:background-color colors/magenta-opa-40
```

Comment:
this is where you should be using the custom-color approach 
so it's better here to do:
```
(defn top-background-view [customization-color]
  {:background-color (colors/custom-color customization-color 50 40)
 ```

Reply:
Thanks. Changed all its appearance.